### PR TITLE
Various little code cleanup

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/xrEngine/xr_ioc_cmd.cpp [hunk 1]
```diff
@@ -246,10 +246,10 @@ void CCC_LoadCFG::Execute(LPCSTR args)
 
     FS.update_path(cfg_full_name, "$app_data_root$", cfg_name);
 
-    if (NULL == FS.exist(cfg_full_name))
+    if (false == FS.exist(cfg_full_name))
```
Comment (Xottab-DUTY) [inline src/xrEngine/xr_ioc_cmd.cpp]:
```suggestion
    if (!FS.exist(cfg_full_name))
```

File: src/xrEngine/xr_ioc_cmd.cpp [hunk 2]
```diff
@@ -246,10 +246,10 @@ void CCC_LoadCFG::Execute(LPCSTR args)
...
         FS.update_path(cfg_full_name, "$fs_root$", cfg_name);
 
-    if (NULL == FS.exist(cfg_full_name))
+    if (false == FS.exist(cfg_full_name))
```
Comment (Xottab-DUTY) [inline src/xrEngine/xr_ioc_cmd.cpp]:
```suggestion
    if (!FS.exist(cfg_full_name))
```

File: src/xrEngine/xr_ioc_cmd.cpp (hunk 1 above)
Comment (devnexen) [reply src/xrEngine/xr_ioc_cmd.cpp]: Yes at least it s more consistent too.

File: src/xrCore/Compression/SubAlloc.hpp [hunk 3]
```diff
@@ -63,7 +63,7 @@ inline void BLK_NODE::insert(void* pv, int NU)
 {
     MEM_BLK* p = (MEM_BLK*)pv;
     link(p);
-    p->Stamp = ~0UL;
+    p->Stamp = ~0U;
```
Comment (Xottab-DUTY) [inline src/xrCore/Compression/SubAlloc.hpp]: DWORD is `unsigned long` on Windows, so `UL` is correct.

**Comment by Xottab-DUTY** [comment]
Thank you!

**Pull request merged**